src/psyclone/tests/dynamo0p3_test.py

```diff
@@ -69,15 +69,6 @@
 
 
 # tests
```

Comment:
Porbably not part of your changes but hey, while we're here :-)

C: 46, 0: third party import "from fparser import api as fpapi" should be placed before "from dynamo0p3_build import Dynamo0p3Build" (wrong-import-order)
C: 46, 0: Imports from package fparser are not grouped (ungrouped-imports)
W:5068, 4: Unused variable 'generated_code' (unused-variable)
W:5087, 4: Unused variable 'generated_code' (unused-variable)
C:1283, 0: Unnecessary parens after 'assert' keyword (superfluous-parens)

Reply:
Done.